doc/man3/CONF_modules_free.pod

```diff
@@ -12,11 +12,10 @@ OpenSSL configuration cleanup functions
  void CONF_modules_finish(void);
  void CONF_modules_unload(int all);
 
-Deprecated:
+Deprecated since OpenSSL 1.1.0, can be hidden entirely by defining
+B<OPENSSL_API_COMPAT>, see L<openssl_user_macros(7)>:
```

Comment:
The formulation is a bit ambiguous IMO, because `#define`ing the constant is not enough. How about something like

>...by defining B<OPENSSL_API_COMPAT=1.1.0> or a higher value, see...

Reply:
Ah, good point

Reply:
The value you showed isn't quite right, though.  I hope that the text I just added is good enough, though

Reply:
> The value you showed isn't quite right, though.

Stupid question: what is the correct value then?